src/components/whiteframe/demoDirectiveUsage/style.css

```diff
@@ -0,0 +1,45 @@
+div[md-whiteframe] {
```

Comment:
*  Attribute selectors are slow in IE.
* This limits the use of `md-whiteframe` to divs ONLY. Should be `md-white-frame`